modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -11502,6 +11482,10 @@ paths:
                   name: fields
                   schema:
                       type: string
+                - in: query
```

Comment:
@sergiojimcos it's confusing that the commits says you're deleting fields but then you add others. It would be much clearer if you deleted the fields in a commit, and added the new ones in another commit that explains why they need to be added.